proto/viam/service/slam/v1/slam.proto

```diff
@@ -123,6 +145,30 @@ message GetInternalStateRequest {
 }
 
 message GetInternalStateResponse {
-  // Internal state of the SLAM algorithm required to continue mapping/localization
+  // A chunk of the internal state of the SLAM algorithm required to continue
+  // mapping/localization
   bytes internal_state = 1;
 }
+
+message GetPointCloudMapStreamRequest {
+  // Name of slam service
+  string name = 1;
+}
+
+message GetPointCloudMapStreamResponse {
+  // pointclouds are returned in chunks as a set of bytes which can be
+  // aggrigated into the standard PCD format
+  // https://pointclouds.org/documentation/tutorials/pcd_file_format.html
+  bytes point_cloud_pcd_chunk = 1;
+}
+
+message GetInternalStateStreamRequest {
+  // Name of slam service
+  string name = 1;
+}
+
+message GetInternalStateStreamResponse {
+  // Chunk of the internal state of the SLAM algorithm required to continue
+  // mapping/localization
+  bytes internal_state_chunk = 1;
```

Comment:
I added the `_chunk` suffix to this member for the same reasons mentioned here: https://github.com/viamrobotics/api/pull/196/files#r1108841266

Reply:
Good idea, can you mention it in the doc you are creating that this additional API change exists? I don't think we need to hold off on starting but I want to make sure Tess is aware.

Reply:
I like making this explicitly know that the message bytes are chunks and not the full PCD